docs/tutorials/educators/neutral_atom.ipynb

```diff
@@ -89,703 +117,804 @@
+    "id": "533125e85e19"
    },
    "source": [
-    "### Moment/Circuit Rules\n",
-    "\n",
-    "Now that we know how to parametarize a NeutralAtomDevice, we can discuss some examples of valid and invalid moments and circuits. Each operation in a moment is treated as if they are performed simultaneously. This leaves us with a few constraints we need to abide by.\n",
-    "\n",
-    "\n",
-    "1.   We need to respect the maximum number of parallel gates for any gate type as discussed above\n",
-    "2.   All instances of gates in the same category in the same moment must be identical\n",
-    "3.   Since controlled gates make use of all types of light used to make gates, controlled gates cannot be applied in parallel with other gate types\n",
-    "4.   Qubits acted on by different controlled gates in parallel must be farther apart than control_radius so that the entanglement mechanism doesn't cause the gates to interfere with one another\n",
-    "5.   Measurements must be terminal\n",
-    "\n",
+    "Multiple controls are allowed as long as every pair of atoms (qubits) acted on by the controlled gate are close enough to each other. We can see this by using the `validate_operation` (or `validate_circuit`) method, as follows."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "id": "59ce97fd61ec"
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"Examples of valid and invalid multi-controlled gates.\"\"\"\n",
+    "# This TOFOLLI is valid because all qubits involved are close enough to each other.\n",
+    "valid_tofolli = cirq.TOFFOLI.on(cirq.GridQubit(0, 0), cirq.GridQubit(0, 1), cirq.GridQubit(0, 2))\n",
+    "neutral_atom_device.validate_operation(valid_tofolli)\n",
     "\n",
+    "# This TOFOLLI is invalid because all qubits involved are not close enough to each other.\n",
+    "invalid_tofolli = cirq.TOFFOLI.on(cirq.GridQubit(0, 0), cirq.GridQubit(1, 0), cirq.GridQubit(0, 2))\n",
     "\n",
-    "---\n",
+    "# Uncommenting raises ValueError: Qubits cirq.GridQubit(1, 0), cirq.GridQubit(0, 2) are too far away.\n",
+    "# neutral_atom_device.validate_operation(invalid_tofolli)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "4ac323a867ad"
+   },
+   "source": [
+    "`NeutralAtomDevice`s do not currently support gates with more than two controls although these are in principle allowed by the physical realizations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {
+    "id": "30d77c374b01"
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"Any gate with more than two controls is invalid.\"\"\"\n",
+    "invalid_gate = cirq.ControlledGate(cirq.TOFFOLI)\n",
     "\n",
-    "Some examples of valid moments are given below. We know the moments we constructed were valid because the circuit allowed us to add them knowing that they needed to pass the neutral_device validation methods."
+    "# Uncommenting raises a ValueError.\n",
```

Comment:
I'd rather put a `try/except` here and print out the error that "as expected we ran into an error: ...` Similar to https://cirq.readthedocs.io/en/latest/docs/protocols.html

Reply:
I like this much better, thanks! Added for each example.